bin/install.sh

```diff
@@ -10,6 +10,12 @@ kitDir=$HOME/.fantastic-kit
 kitConfigDir=$HOME/.config/fantastic-kit
 kitRootSrcDir=$HOME/src/github.com
 
+# check if git is installed
+if ! command -v git > /dev/null; then
```

Comment:
If we are trying to be POSIX compliant then we shouldn't even be using `[[ ]]` at all, since it is a bash extension, `command -v` is also a bash built-in command. As of now we are using `[[ ]]` everyone throughout the codebase 

I think the bigger question here is that do we want to be POSIX compliant? Which means we need to be able to execute inside `sh`. In our current README, we tell our users to install `bash`, which is perfectly fine for being not POSIX compliant.

Also `if ;` and `if [[ ]];` have different semantics and one is using the return code of the command whereas the other one is using the stdout with the `test` command. I don't think we should use `if [[ ]];` simply just for the sake of consistency. 


cc @fan70m

Reply:
We reply on this being used with zsh. Our install scripts write out to `$HOME/.zshrc`, our docker image install zsh, oh-my-zsh and sets default shell to zsh.